### ISSUE TITLE
Downloads Failing

### ISSUE THREAD

**Issue by starcrash-13** [pranjalagg/spdl#30]
Almost all attempts to download are throwing a 'link' error, everything seems to be working fine on spotifydown.com

**Comment by p42rthicle** [comment]
Yeah, same

**Comment by mdeeter** [comment]
it looks like they added CloudFare in front of the download option... it requires a token now

**Comment by pranjalagg** [comment]
Yes, @mdeeter you are right, they have added protection for their API. I don't know if there's any way to bypass that.
<img width="451" alt="image" src="https://github.com/user-attachments/assets/2bfc4cb5-97b3-4d8f-8fe4-eff773f1e675">

**Comment by starcrash-13** [comment]
It still works periodically though, and the spotifydown website is working...?

**Comment by mdeeter** [comment]
Spotifydown webiste is working because they control the token.  I've tried it a few times each of these past few days and it hasn't worked since they've required the token in the /download call.

**Comment by pranjalagg** [comment]
Yeah, it even works for me periodically, but I guess this is just short lived. I wouldn't be surprised if it stops working at all sometime in the future.

**Comment by rocko-mx** [comment]
Great idea, sadly is it working no more :-1:

**Comment by pranjalagg** [comment]
The README file has now been updated with the workaround. It involves manually copying the token and pasting it to the program. If any of you feel there could be a better way to solve this, please feel free to reopen this issue or create a new one.
Thanks!
